### PR TITLE
Added file containing question about fake Tor Browsers

### PR DESCRIPTION
**Pull request by MacTavish-69**
Added a new folder _"are-there-any-paid-versions-of-tor-browser"_
Added _"contents.lr"_ file in the folder.
[Issue Link](https://dip.torproject.org/torproject/web/support/issues/69)

### CODE REVIEW

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr [hunk 1]
```diff
@@ -0,0 +1,17 @@
+_model: question
+---
+title: Are there any paid versions of Tor Browser?
+---
+seo_slug: are-there-any-paid-versions-of-tor-browser
+---
+description:
+
+No, Tor Browser is an open source software and it is FREE. Any browser forcing you to pay or donate and is claiming to be
```
Comment (gusgustavo) [inline content/are-there-any-paid-versions-of-tor-browser/contents.lr]: Remove the caps on FREE and FAKE.

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+No, Tor Browser is an open source software and it is FREE. Any browser forcing you to pay or donate and is claiming to be
+Tor browser is FAKE. To make sure you are downloading the right tor browser visit our [download page](https://www.torproject.org/download/) .
+After downloading, you can make sure that you have the official version of Tor browser by following [this process](https://support.torproject.org/tbb/how-to-verify-signature/) .
+If you are not able to access [our website](torproject.org) then visit [censorship section](https://support.torproject.org/censorship/censorship-3/)
+to get information about alternate way of downloading Tor.
+
+### Report Fake Tor Browsers
+
+You can report fake Tor Browsers on frontdesk@torproject.org
```
Comment (gusgustavo) [inline content/are-there-any-paid-versions-of-tor-browser/contents.lr:15]: We don't need a new subtitle here if it's only one phrase.

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr [hunk 3]
```diff
@@ -0,0 +1,17 @@
+_model: question
+---
+title: Are there any paid versions of Tor Browser?
+---
+seo_slug: are-there-any-paid-versions-of-tor-browser
+---
+description:
+
+No, Tor Browser is an open source software and it is FREE. Any browser forcing you to pay or donate and is claiming to be
+Tor browser is FAKE. To make sure you are downloading the right tor browser visit our [download page](https://www.torproject.org/download/) .
```
Comment (gusgustavo) [inline content/are-there-any-paid-versions-of-tor-browser/contents.lr]: Fix to Tor Browser here.

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+---
+title: Are there any paid versions of Tor Browser?
+---
+seo_slug: are-there-any-paid-versions-of-tor-browser
+---
+description:
+
+No, Tor Browser is an open source software and it is FREE. Any browser forcing you to pay or donate and is claiming to be
+Tor browser is FAKE. To make sure you are downloading the right tor browser visit our [download page](https://www.torproject.org/download/) .
+After downloading, you can make sure that you have the official version of Tor browser by following [this process](https://support.torproject.org/tbb/how-to-verify-signature/) .
```
Comment (gusgustavo) [inline content/are-there-any-paid-versions-of-tor-browser/contents.lr]: Fix Tor Browser here too.

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+If you are not able to access [our website](torproject.org) then visit [censorship section](https://support.torproject.org/censorship/censorship-3/)
```
Comment (gusgustavo) [inline content/are-there-any-paid-versions-of-tor-browser/contents.lr]: add https:// to torproject.org

**Review by gusgustavo — CHANGES_REQUESTED** [review]
Good stuff!

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr (hunk 1 above)
Comment (MacTavish-69) [reply content/are-there-any-paid-versions-of-tor-browser/contents.lr]: fixed

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr (hunk 2 above)
Comment (MacTavish-69) [reply content/are-there-any-paid-versions-of-tor-browser/contents.lr:15]: fixed

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr (hunk 4 above)
Comment (MacTavish-69) [reply content/are-there-any-paid-versions-of-tor-browser/contents.lr]: fixed

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr (hunk 5 above)
Comment (MacTavish-69) [reply content/are-there-any-paid-versions-of-tor-browser/contents.lr]: fixed. All good ..

**Comment by MacTavish-69** [comment]
I believe this PR is ready to be merged.

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr [hunk 6]
```diff
@@ -0,0 +1,15 @@
+_model: question
+---
+title: Are there any paid versions of Tor Browser?
+---
+seo_slug: are-there-any-paid-versions-of-tor-browser
+---
+description:
+
+No, Tor Browser is an open source software and it is free.
+Any browser forcing you to pay or donate and is claiming to be Tor browser is fake.
```
Comment (emmapeel2) [inline content/are-there-any-paid-versions-of-tor-browser/contents.lr]: Please write it Tor Browser, capital B.

**Review by emmapeel2 — COMMENTED** [review]
Please add a shorter folder name

File: content/are-there-any-paid-versions-of-tor-browser/contents.lr (hunk 6 above)
Comment (MacTavish-69) [reply content/are-there-any-paid-versions-of-tor-browser/contents.lr]: Please suggest name, I used the existing conventions in project and tried to make it self explanatory. If there is no technical issues of a bit longer name then I think we keep it this way :)

**Pull request merged**